pkg/operator/config_reloader.go

```diff
@@ -249,4 +251,44 @@ func CreateConfigReloader(name string, options ...ReloaderOption) v1.Container {
 			},
 		},
 	}
+
+	if !configReloader.runOnce && configReloader.config.EnableProbes {
+		c = addProbes(c)
+	}
+
+	return *c
+}
+
+func addProbes(c *v1.Container) *v1.Container {
```

Comment:
IMHO, I think it is best not to change the container that is passed in as input unless there is a strong reason to. 
```
func addProbes(c v1.Container) v1.Container {
  c.LivenessProbe = livenessProbe
  c.ReadinessProbe = readinessProbe
  return c
```

Reply:
Thinks for U review.   Done